# Request
sensor_msgs/LaserScan laser_data
---
# Response
string direction